src/modules/mc_pos_control/mc_pos_control_main.cpp

```diff
@@ -568,6 +568,7 @@ MulticopterPositionControl::parameters_update(bool force)
 		param_get(_params_handles.thr_min, &_params.thr_min);
 		param_get(_params_handles.thr_max, &_params.thr_max);
 		param_get(_params_handles.thr_hover, &_params.thr_hover);
+		_params.thr_hover = math::constrain(_params.thr_hover, _params.thr_min, _params.thr_max);
 		param_get(_params_handles.alt_ctl_dz, &_params.alt_ctl_dz);
```

Comment:
I just wanted to make sure the user is not able to set 10'000% hover throttle because the integrator would not be able to compensate for that and the vehicle would fly to "infinite" altitude. Normal values like 100% will not lead to nice takeoffs but should not cause any flyaway or crash. This matches the level of checks that the rest of the code has.